### DISCUSSION TITLE
Publish VM images to Azure Marketplace

### DISCUSSION THREAD

**Discussion by gaui** [actions/virtual-environments#7482] [Ideas]
Instead of having to build the images and publishing them ourselves, would it be possible to publish these VM images to the Azure Marketplace?

**Comment by markatky46** [comment]
They can't do this as it would be classed as distributing licensed software.

**Reply by erik-bershel** [reply]
The answer exact.

**Reply by pkit** [reply]
@markatky46 Which licensed software exactly will be distributed in Ubuntu 22.04 image?
